templates/index.html

```diff
@@ -0,0 +1 @@
+Sample page in case required.
```

Comment:
It is better to make this a comment.

Reply:
but this *should* be replaced with a proper page anyway. is there a point?

Reply:
Just a little thing like formatting. Before it is replaced.